components/Admin/index.vue

```diff
@@ -134,8 +134,10 @@ export default {
     searchTitle: debounce(function (value) {
       if (value.length > 2) {
         this.query.q = value
-      } else {
+        this.$fetch()
+      } else if (value.length === 0) {
         this.query.q = null
+        this.$fetch()
```

Comment:
ask

just curious, so the `query.q` value won't be updated when the `value.length` is 1 and 2 ?

Reply:
yes, it wont until it has minimum 3 char